roles/external_secrets_operator/vars/main.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+_external_secrets_operator_helm_values:
+  global:
+    nodeSelector: "{{ external_secrets_operator_node_selector }}"
+  image:
+    repository: "{{ external_secrets_operator_image | atmosphere.common.docker_image('name') }}"
+    tag: "{{ external_secrets_operator_image | atmosphere.common.docker_image('tag') }}"
+  installCRDs: true
+  leaderElect: true
```

Comment:
Since this defaults to `false`, and we're only running a single replica. Should we drop this to go to default, or go back to true but add replicas?

Reply:
From my experience is better to set this to `true` because in case someone sets replicas to 1+ - 100% will forget turn this to `true`.